fix FloatRange deserialize xml errors

emulatethis777 opened the pull request.

TheLoneTec commented:
ok, hold on, just checked this, all of vanilla uses this format, im not sure what caused your errors, but im reverting this for now, could you explain more your reasoning for this? especially if you did get errors, to share the logs

emulatethis777 commented:
Strange, but I don't get these errors anymore.
Maybe it was because of some of my mods. Last night, I updated and removed a few of them.
The errors were like this
`XML error: 50~800 doesn't correspond to any field in type FloatRange`
Anyway, thank you for your time. Next time I'll try to reproduce errors with 'vanilla' HSK.
